examples/platform/silabs/efr32/rs911x/sl_wifi_if.cpp

```diff
@@ -0,0 +1 @@
+../../SiWx917/SiWx917/sl_wifi_if.c
```

Comment:
Broken symlink ? if the file was change for .cpp extension

Reply:
Symlink fixed.